### PR TITLE
[Android] Add Multi Device Example - @open sesame 8/01 10:13

### PR DESCRIPTION
**Pull request by jijoongmoon**
# PR Description

This is the applicaiton for the demonstrate nnstreamer with multi
device. In order to execute this application, you may need three
devices, one for the camera input and prepocessing, one for the object
detection and the last for the pose estimation.

Once it gets the input from camera, it do the preprocessing. and after
that first device send the data to the second device. In second
device, object detection is conducted using tensorflow lite and if
there is person, then post processing is executed. after that the data
send to the last device, and the pose estimaition is done in third
device.

In order to transfer the data among devices, tcpclientsrc and
tcpserversink is used.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: android/multi_device_shared_lib/jni/NNStreamerMultiDevice.c [hunk 1]
```diff
@@ -0,0 +1,1028 @@
...
+      str_pipeline =
+          g_strdup_printf
+          ("tcpclientsrc host=%s port=%d ! image/jpeg, framerate=30/1 ! jpegparse ! jpegdec ! videoconvert ! "
+          "tee name=t t. ! queue ! videoconvert ! cairooverlay name=res_overlay ! glimagesink sync=false "
+          "t. ! queue leaky=2 max-size-buffers=2 ! videoscale ! video/x-raw, width=192, height=192, format=RGB ! "
+          "tensor_converter ! tensor_transform mode=typecast option=float32 qos=true ! "
+          "tensor_filter framework=tensorflow-lite model=/sdcard/nnstreamer/tflite_model/detect_pose.tflite qos=true ! "
+          "tensor_sink name=res_sink sync=false ", data->ipaddr, data->portnum);
+      break;
+    defalut:
```
Comment (jaeyun-jung) [inline android/multi_device_shared_lib/jni/NNStreamerMultiDevice.c]: default  maybe

Comment (jijoongmoon) [reply android/multi_device_shared_lib/jni/NNStreamerMultiDevice.c]: Yes. Thanks.

**Comment by jijoongmoon** [comment]
It is ready for further review. Thanks.

File: android/multi_device_shared_lib/jni/NNStreamerMultiDevice.c [hunk 2]
```diff
@@ -0,0 +1,1029 @@
...
+    case 0:
+      color = g_strdup ("#0000FF");
+      break;
+    case 1:
+      color = g_strdup ("#008000");
+      break;
+    case 2:
+      color = g_strdup ("#FF0000");
+      break;
+    defalut:
```
Comment (jaeyun-jung) [inline android/multi_device_shared_lib/jni/NNStreamerMultiDevice.c]: build error maybe 'defalut'
others ok.

Comment (jijoongmoon) [reply android/multi_device_shared_lib/jni/NNStreamerMultiDevice.c]: done.

**Pull request merged**